README.md

```diff
@@ -134,6 +134,7 @@ The notes are sorted alphabetically, major (upper case) notes before minor
 | Fan/LED controller | [NZXT RGB & Fan Controller](docs/nzxt-hue2-guide.md) | USB HID | <sup>_h_</sup> |
 | Fan/LED controller | [NZXT Smart Device](docs/nzxt-smart-device-v1-guide.md) | USB HID | <sup>_h_</sup> |
 | Fan/LED controller | [NZXT Smart Device V2](docs/nzxt-hue2-guide.md) | USB HID | <sup>_h_</sup> |
+| Fan/AIO controller | [NZXT H1 V2 Smart Device](docs/nzxt-hue2-guide.md) | USB HID | |
```

Comment:
Nit: let's consistently call it `NZXT H1 V2`.  This also matches what was chose for the Obsidian 1000D.